src/facade/contentFetcherFacade.ts

```diff
@@ -64,6 +75,7 @@ export class ContentFetcherFacade {
             preferredLocale: options.preferredLocale,
             context: this.contextFactory.createContext(options.attributes),
             timeout: options.timeout,
+            ...(preferredLocale !== undefined ? {preferredLocale: preferredLocale} : {}),
```

Comment:
Is this necessary? We are not treating a missing property and an explicit `undefined` as different (and I don't think we should).

This would work the same, right?
```suggestion
            preferredLocale: preferredLocale,
```

Reply:
I always avoid undefined values since they lead to silly bugs.